packages/@okta/vuepress-site/docs/reference/api/factor-admin/index.md

```diff
@@ -54,8 +54,8 @@ The following providers are supported:
 | ---------- | ----------------------------- |
 | `GOOGLE`   | Google Integration            |
 | `OKTA`     | Okta                          |
-| `RSA`      | RSA SecurID Integration       |
 | `SYMANTEC` | Symantec VIP Integration      |
+<!--- | `RSA`      | RSA SecurID Integration       | --->
```

Comment:
Same as previous comment.

Reply:
Same comment as above. Looking for final resolution. We started this process because I was told RSA is broken, never worked and should be removed. While finalizing that I asked 'Are you sure'. Still getting conflicting responses!

Reply:
No problem! Just didn't want to hold things up.